src/app/components/arlas-wui-root/arlas-wui-root.component.scss

```diff
@@ -227,11 +241,11 @@
 
 .app-container-reduce-analytics,
 .app-container-reduce {
-  .arlas-map-action,
-  .arlas-map-settings {
-    right: calc($result-list-width + $sm-spacing);
+  ::ng-deep.arlas-map-action,
```

Comment:
shouldn't this classes go the the new component ?

Reply:
That is a good question. All those considerations about where to put the map actions, settings and AOI dimensions, are, to me, only relevant when we load the arlas-wui-root component. Since the goal is to provide externalisation of the components, it would mean adding unnecessary css rules in the case of a multi window consultation of ARLAS. It is something that could be discussed however, if it is found to be clearer to move those considerations to the map component itself